tests/python/integration/test_scenario.py

```diff
@@ -0,0 +1,25 @@
+from dku_plugin_test_utils import dss_scenario
+
+
+def test_run_read_zendesk_groups(user_dss_clients):
+    dss_scenario.run(user_dss_clients, 'PLUGINTESTZENDESK', 'run_read_zendesk_groups')
+
+
+def test_run_read_zendesk_incremental(user_dss_clients):
+    dss_scenario.run(user_dss_clients, 'PLUGINTESTZENDESK', 'run_read_zendesk_incremental')
+
+
+def test_run_read_zendesk_organizations(user_dss_clients):
+    dss_scenario.run(user_dss_clients, 'PLUGINTESTZENDESK', 'run_read_zendesk_organizations', user="user1")
```

Comment:
Is is possible to change `user1` to new dku34 user named `data_scientist_1`? Thanks!
```suggestion
    dss_scenario.run(user_dss_clients, 'PLUGINTESTZENDESK', 'run_read_zendesk_organizations', user="data_scientist_1")
```